typings/__tests__/index.test.tsx

```diff
@@ -52,30 +50,24 @@ const getByDisplayValueString: ReactTestInstance = tree.getByDisplayValue(
 const getByDisplayValueRegExp: ReactTestInstance = tree.getByDisplayValue(
   /value/g
 );
-const getByProps: ReactTestInstance = tree.getByProps({ value: 2 });
+const getByProps: ReactTestInstance = tree.UNSAFE_getByProps({ value: 2 });
 const getByTestId: ReactTestInstance = tree.getByTestId('test-id');
 const getAllByTestId: ReactTestInstance[] = tree.getAllByTestId('test-id');
-const getAllByNameString: Array<ReactTestInstance> = tree.getAllByName('View');
-const getAllByNameConstructor: Array<ReactTestInstance> = tree.getAllByName(
-  View
+const getAllByType: Array<ReactTestInstance> = tree.UNSAFE_getAllByType(View);
+const getAllByTypeWithRequiredProps: Array<ReactTestInstance> = tree.UNSAFE_getAllByType(
+  ElementWithRequiredProps
 );
-const getAllByType: Array<ReactTestInstance> = tree.getAllByType(View);
-const getAllByTypeWithRequiredProps: Array<
-  ReactTestInstance
-> = tree.getAllByType(ElementWithRequiredProps);
 const getAllByTextString: Array<ReactTestInstance> = tree.getAllByText(
   '<View />'
 );
 const getAllByTextRegExp: Array<ReactTestInstance> = tree.getAllByText(/Text/g);
-const getAllByProps: Array<ReactTestInstance> = tree.getAllByProps({
+const getAllByProps: Array<ReactTestInstance> = tree.UNSAFE_getAllByProps({
```

Comment:
Now that I look at TypeScript, can we add something like `@deprecated` to TSDoc of removed functions? Will that work? It would provide some nice feedback when migrating :)

Reply:
Added `@deprecated` for all removed or renamed functions